### DISCUSSION TITLE
2.2.1

### DISCUSSION THREAD

**Discussion by olavopeixoto** [olavopeixoto/plugin.video.brplay#181] [Announcements]
Updated Globoplay API
Added multiple regions (Europe, Portugal, USA) support
Added more Globo affiliates
Fixed Search feature
Added support for VIX TV
Removed Telecine Play (Moved to Globoplay)
Removed Sexyhot Play

<hr /><em>This discussion was created from the release <a href='https://github.com/olavopeixoto/plugin.video.brplay/releases/tag/2.2.1'>2.2.1</a>.</em>

**Comment by marcusmuller** [comment]
Faltou o Canadá!

**Comment by olavopeixoto** [comment]
Canada e o mesmo do EUA.

**Comment by marcoscamurati** [comment]
Boa tarde,
Não consegui instalar essa última vrs no Matrix, não funciona?
